kie-server-parent/kie-server-remote/kie-server-rest/kie-server-rest-jbpm/src/main/java/org/kie/server/remote/rest/jbpm/RuntimeDataResource.java

```diff
@@ -768,7 +742,7 @@ public Response getTasksByVariables(@Context HttpHeaders headers,
         }
     }
 
-    @ApiOperation(value = "Queries processes by variables and tasks")
+    @ApiOperation(value = "Queries processes by variables and tasks", response = ProcessInstanceCustomVarsList.class)
```

Comment:
For consistency, maybe we can use this PR to include a 200 response code for this method.

Reply:
200 is the default, so it is implicit